cypress/e2e/rich-text/document-mocks/paragraphWithoutFormattings.js

```diff
@@ -34,7 +34,7 @@ export default {
           marks: [],
           nodeType: 'text',
           value:
-            " is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. ",
+            " is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. ",
```

Comment:
🤷🏻‍♂️

Reply:
used to be a `&nbsp;`

Reply:
Since upgrading `&nbsp;` have been stripped out when pasting in microsoft word text

Reply:
https://github.com/udecode/plate/blob/18d99ddfde960744cc19d812cdf0690a378caaee/packages/core/src/plugins/html-deserializer/utils/cleanHtmlTextNodes.ts#L32C9-L35C10